### ISSUE TITLE
WiFi ESP8266-01S can't connect with SKR MINI E3 v2 + TFT35 E3 V3

### ISSUE THREAD

**Issue by bubulgum** [bigtreetech/BIGTREETECH-TFT35-V3.0#133]
I have installed software to wifi,and if connect to TFT i can see web interface in browers (have uploaded index.html.gz from here https://github.com/bigtreetech/BIGTREETECH-SKR-PRO-V1.1/tree/master/firmware/esp-01s) and last version of ESP3D with env:esp01s_160mhz-2.7.4 with USE_SERIAL 0
![image](https://user-images.githubusercontent.com/21036145/125833303-7660616e-108f-489c-8757-cc3b87324462.png)
But still can't see printer...even on display M117 don't show connecting message.
Looks like wifi module inactive with Firmware.
TFT with SKR connected to EXP3-CR10 and TFT cables.
Alos have tried in TFT config.ini
`serial_always_on:1 `
In Marlin i have enabled 
```
#define SERIAL_PORT -1
#define SERIAL_PORT_2 2

//also have tried

#define SERIAL_PORT 3
#define SERIAL_PORT_2 -1

and etc...
```
[Marlin.zip](https://github.com/bigtreetech/BIGTREETECH-TFT35-V3.0/files/6825275/Marlin.zip)
[TFT Config.zip](https://github.com/bigtreetech/BIGTREETECH-TFT35-V3.0/files/6825284/TFT.Config.zip)
[ESP3D config.zip](https://github.com/bigtreetech/BIGTREETECH-TFT35-V3.0/files/6825403/ESP3D.config.zip)

**Comment by radek8** [comment]
If you connect the WiFi module to the TFT, you have a configuration error.
The serial port for WiFi is disabled.

![image](https://user-images.githubusercontent.com/46979052/125844539-0ba34656-8e0c-4307-af2c-7ec1719d84ac.png)

**Comment by bubulgum** [comment]
> If you connect the WiFi module to the TFT, you have a configuration error.
> The serial port for WiFi is disabled.

Thank you for you replies.

I have enabled:
multi_serial: 1
And Marlin have:
#define SERIAL_PORT -1
#define SERIAL_PORT_2 2

But still no emotion.

**Comment by radek8** [comment]
Do you have your Wifi access point correctly defined on the wifi module?
Do you have a defined baud rate?

![image](https://user-images.githubusercontent.com/46979052/125851820-6d18f9a4-0dbe-449d-81a2-d72b9bc7a93b.png)

**Comment by bubulgum** [comment]
yes, i can connet to the web interface, set up correct Baud Rate.
Looks like Wifi don't have connection with printer, only as power, but not with firmware.
I'm thinking htat problem with Serial`s...but can't figure out how to check.

**Comment by radek8** [comment]
Your speed is 115200
do you have set

**Comment by radek8** [comment]
set in config.ini
serial_always_on:1

**Comment by bubulgum** [comment]
![image](https://user-images.githubusercontent.com/21036145/125915853-b018a70c-9f83-4c35-841b-889a652a52e9.png)


> Your speed is 115200
> do you have set

i have tried 115200, then 250000..nothing.

> set in config.ini
> serial_always_on:1

have set, i think problem with serials, but i don't know where to check exactly right connections.

**Comment by radek8** [comment]
> 
> i have tried 115200, then 250000..nothing.
> 

You must not rehearse, you must know what you are setting.

There are a lot of parameters, if you try parameters that are clear, it will take you a long time :-(

**Comment by bubulgum** [comment]
can not find where is no connection. spped rate 250000, have enabled ports. but same issue. is it possible to test wifi connection by firmware? because i can connect to wifi through IP, so it is mean wifi connected right and working. but there is no connection with firmware.
Can it be problem because my firmware is Marlin but modify by KAD? is it important?
[configs.zip](https://github.com/bigtreetech/BIGTREETECH-TFT35-V3.0/files/6911113/configs.zip)

**Comment by radek8** [comment]
if you have WiFi connected to TFT and TFT communicates with SKR, WIFi should also communicate.
Must have WiFi to TFT serial port enabled (you have it)
The same communication speed must be selected.
If it still does not communicate there, you have Wifi badly connected, or some pin on the serial WiFi connector is damaged.

**Comment by radek8** [comment]
I have no experience with the KAD configuration, I cannot judge. But I'm quite confusing for me.

Send a photo of the connection

**Comment by bubulgum** [comment]
I found what was problem...i have uploaded old index.html.gz
now everything wokrs well with this one [https://github.com/luc-github/ESP3D-WEBUI](url)
And also before i have uploaded ESP8266, but i need ESP8266-1S [https://github.com/luc-github/ESP3D](url)
and also i have uploaded last update for TFT and config file
Thank you for help
![Screenshot_1](https://user-images.githubusercontent.com/21036145/127740885-8b38a56a-f4bd-48ae-904d-22438f598319.png)

**Comment by radek8** [comment]
The web interface of the wifi adapter doesn't suit me very well.
I connected the printer's WIFI adapter as a virtual serial port and I connect to the printer via the Pronterface wifi program.
You can try it, it's functional :-)

**Comment by KingBain** [comment]
Just went through this. with SKR Mini E3 v3 and ESP8266-01(clone modules); **not the S variant..** 

I wired everything to the TFT port on the main board 
![image](https://user-images.githubusercontent.com/367922/209756684-951960e0-e732-40e4-bb55-68bf21a04a80.png)


I know the chip doesn't like 5v, but that's what I normally feed it and they work fine. 

In my marlin config. 

I haven't changed much, but this is what I'm using 

#define SERIAL_PORT 2
#define BAUDRATE 115200
#define SERIAL_PORT_2 -1
